components/clarity-repl/src/repl/session.rs

```diff
@@ -1427,6 +1472,81 @@ mod tests {
         let mut session = Session::new(settings);
         session.start().expect("session could not start");
 
+        session.handle_command("::set_epoch 3.0");
```

Comment:
This replaces in the test `evaluate_at_block`, which was testing `at-block` (and apparently a few other things, looks like it was really a "unit" test).
But the new test you introduced tests something else, although I'm not sure why.
In think that `evaluate_at_block` should remain untouched and you should add an other test

Reply:
> the original is now below called evaluate_at_block_pre_epoch3

Yes, but I think it should remain `evaluate_at_block` since it tests the `at-block` clarity method, and isn't influenced by the epoch things.
Or, if we want to make sure it still works as expected in epoch3.0, I would add another test `evaluate_at_block_in_epoch_3_0` instead of changing what already existed. 


> what should this epoch 3 test be testing

idk, mayne it's redundant